src/unitxt/metric_utils.py

```diff
@@ -320,6 +320,30 @@ def prepare(self):
 )
 
 
+class EvaluationResults(list):
+    @property
+    def score(self):
+        return self[0]["score"]["global"]
+
+    @property
+    def groups(self):
+        if "groups" not in self[0]["score"]:
+            raise ValueError("Groups scores not found try using group_by in the recipe")
+        return self[0]["score"]["groups"]
+
+    @property
+    def subsets(self):
+        if "subsets" not in self[0]["score"]:
+            raise ValueError("Subsets scores not found try using Benchmark")
```

Comment:
We should have a UnitxtError("No subsets were defined using using the Benchmark in the recipe. For more information ....) and point the
documentation